Makefile

```diff
@@ -406,7 +406,7 @@ test-race : GO_TEST_EXTRA_ARGS=-race
 # And so on.
 test : fmt
 	(unset GIT_DIR; unset GIT_WORK_TREE; \
-	$(GO) test $(GO_TEST_EXTRA_ARGS) $(addprefix ./,$(PKGS)))
+	$(GO) test -count=1 $(GO_TEST_EXTRA_ARGS) $(addprefix ./,$(PKGS)))
```

Comment:
Just for my enlightenment, is there a place where the caching is not doing the right thing here?

Reply:
No, the caching is doing the right thing -- albeit, if you're invoking the wrong `git-lfs`, it caches those results, hence my confusion in thinking there might be some transient behaviour in the tests.  With the other changes in this PR, that _should_ stop being an issue, so one could leave this as-is.

But, speaking just for myself, I expect a test suite to run fully every time, at least if I'm using a `make test` kind of invocation.  I realize Go likes to cache its test results, although only when using "package list mode".  For my part, if I'm using a wrapper like `make test` I may not realize what's going on under the hood, and if it's being invoked in a CI context, I'd prefer to know it is forcing a full redo on all the tests.

Anyway, that's why I figured this was a reasonable change; someone can always use `go test ./...` if they want the default Go behaviour.  But I can live without it too.  :-)